spec/controllers/expenses_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe ExpensesController, type: :controller do
+  include Devise::Test::ControllerHelpers
+  let(:user) { create(:user) }
+  let(:category) { create(:category, user:) }
+
+  before do
+    sign_in user
+  end
+
+  describe 'GET #new' do
+    it 'assigns a new expense to @expense' do
```

Comment:
- [ ] I can see 21 of your test cases passed, good job but I will encourage you to double-check the remaining as they are failing as shown in the image below.
![image](https://github.com/ibugithub/Alien_transction/assets/86200356/dc19e425-9c9d-4ffe-a576-3253ccd29ebf)

Reply:
I think you have some problems with your setup. Try to fix it. All of my tests are passing here in my system.
![image](https://github.com/ibugithub/Alien_transction/assets/61221884/c4d5baec-e33e-4faf-acc1-8ff30f6b8dea)


Check out the video. All of my tests has been passed. 
https://www.loom.com/share/18a2fb96091946cf9d6036ac5a29d242